server/chat/backend/agent/tools/recent_incidents_tool.py

```
@@ -0,0 +1,124 @@
+    "Returns alert titles, affected services, severity, and root cause summaries. "
+    "Available for situational awareness during investigation."
+)
+
+
+def get_recent_incidents(
+    time_window_hours: int = 72,
+    limit: int = 20,
+    service_filter: Optional[str] = None,
+    user_id: Optional[str] = None,
+    incident_id: Optional[str] = None,
+    **kwargs,
```

Ruff (0.15.15)

[warning] Missing type annotation for `**kwargs`

(ANN003)